### DISCUSSION TITLE
github actions build problem

### DISCUSSION THREAD

**Discussion by hsglc** [supabase/supabase#19843] [Questions]
Hey there,

I'm using the github action as deploying to the vercel. I'm using pnpm.
There is a build step in my job, when it's running an error occured. Can you anyone help me to solve that?

my .github/workflow/yaml file:

`name: Vercel Preview Deployment

on:
  push:
    branches: [main]
  pull_request:
    branches: [main]

jobs:
  deployment:
    name: 'Lint, Format, and Build'
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Setup pnpm
        uses: pnpm/action-setup@v2.2.4
        with:
          version: 8.6.3

      - name: Use Node.js 18
        uses: actions/setup-node@v3

        with:
          node-version: '18'
          cache: 'pnpm'

      - name: Install dependencies
        run: pnpm install --frozen-lockfile

      - name: Run Biome.js
        run: pnpm run check

      - name: Build
        run: pnpm run build`
<img width="1453" alt="Screenshot 2023-12-18 at 22 02 50" src="https://github.com/supabase/supabase/assets/74729094/7c15fcca-923e-4618-ab86-fb077e66342f">

**Comment by quinn-dougherty** [comment]
I got the same problem after renaming my [project](https://github.com/quinn-dougherty/rallypoint), when it was working perfectly from the template before the rename. 

I tried loading my `.env` file into github action variables and secrets, no dice.
